src/main/java/org/itsallcode/openfasttrace/Converter.java

```diff
@@ -93,4 +91,14 @@
      *            exporter plug-in)
      */
     void convertToFileInFormat(final Path output, final String format);
+
+    /**
+     * Get the default export output format
+     * 
+     * @return the default output format
+     */
+    public static String getDefaultOutputFormat()
+    {
+        return "specobject";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3400)